docs/admin/eventing/broker-configuration.md

```diff
@@ -183,18 +183,18 @@ metadata:
   labels:
     eventing.knative.dev/release: devel
 data:
-  # Configuration for defaulting Channels that do not specify CRD implementations.
+  # Configuration for defaulting Brokers that do not specify a spec.config or Broker class.
   default-br-config: |
     clusterDefault:
       brokerClass: MTChannelBasedBroker
 ```
-​
+
 ### Configuring the default BrokerClass for namespaces
-​
+
 You can modify the default Broker class for one or more namespaces.
 
 For example, if you want to use a `KafkaBroker` class for all other Brokers created on the cluster, but you want to use the `MTChannelBasedBroker` class for Brokers created in `namespace-1` and `namespace-2`, you would use the following ConfigMap settings:
-​
+
```

Comment:
In case somebody wonders what these seemingly empty changes mean, the doc contained zero-width space characters which are invisible to the naked eye but were displayed by my editor. I removed them.

![image](https://user-images.githubusercontent.com/3299086/132092357-8b5187ac-3599-4d6d-a114-d8d8f2170a7f.png)